src/redux/modules/transactionsList/reducer.js

```diff
@@ -1,6 +1,8 @@
 import { FETCHED } from './actions.js';
 
-const initialState = {};
+const initialState = {
+  transactionsList: [],
```

Comment:
could you rename it to `data`, please?
looks like you have `transactionsList.transactionsList`

so you'll have
```
case FETCHED:
      return {
        ...state,
        data: action.payload,
      };
 ```

Reply:
DOne